### ISSUE TITLE
Doesnt work on Chrome AppImage (Linux)

### ISSUE THREAD

**Issue by auwsom** [yorkxin/copy-as-markdown#102]
Doesnt work on Chrome AppImage (Linux)

It just shows "no title":
[(No Title)](https://chrome.google.com/webstore/detail/copy-as-markdown/fkeaekngjflipcockcnpobkpbbfbhmdn)

I tested it on Firefox on Kubuntu 20.04 and it works fine. I dont know it it just the AppImage or Chrome.

**Comment by yorkxin** [comment]
Hi, do you still experience this issue?

**Comment by auwsom** [comment]
Yeah, actually doesnt work at all right now (nothing copied), but works on firefox.

**Comment by yorkxin** [comment]
Related: #1
